akka-actor-typed/src/main/scala/akka/actor/typed/javadsl/ActorContext.scala

```diff
@@ -98,6 +120,9 @@ trait ActorContext[T] {
    * processing its current message. Nothing happens if the ActorRef does not
    * refer to a current child actor.
    *
+   * *Warning*: This method is not thread-safe and must not be accessed from other threads
```

Comment:
"threads other than" reads better IMO, WDYT? (for all of these)

Reply:
Or just should only be accessed from the actor message processing thread e.g. not ...